_posts/2015-08-19-home-lab-rebuild.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title: "Home Lab Rebuild"
+description: ""
+category:
+tags: []
+---
+{% include JB/setup %}
+
+Recently a hard drive went out on my main home VM Server.  It was in a RAID, so
+there was no data loss.  But as I had to shut everything down to replace 
+hardware (no hot swap...), it makes me think about also maintaining the 
+software stack for the lab.
+
+Currently I have two physical systems that are libvirt/KVM hosts for my 
+virtual machines, including the main firewall and fileserver.  I've had my eye
+on a few newer technologies, so I'd make the lab flexible.
```

Comment:
I updated the language to hopefully make it more clear.  Let me know what
you think.

On Wed, Aug 19, 2015 at 8:40 AM Sarah Miller <notifications@github.com>
wrote:

> In _posts/2015-08-19-home-lab-rebuild.md
> <https://github.com/avrgeek/avrgeek.github.io/pull/3#discussion_r37409123>
> :
>
> > +layout: post
> > +title: "Home Lab Rebuild"
> > +description: ""
> > +category:
> > +tags: []
> > +---
> > +{% include JB/setup %}
> > +
> > +Recently a hard drive went out on my main home VM Server.  It was in a RAID, so
> > +there was no data loss.  But as I had to shut everything down to replace
> > +hardware (no hot swap...), it makes me think about also maintaining the
> > +software stack for the lab.
> > +
> > +Currently I have two physical systems that are libvirt/KVM hosts for my
> > +virtual machines, including the main firewall and fileserver.  I've had my eye
> > +on a few newer technologies, so I'd make the lab flexible.
>
> This sentence feels awkward, not sure what you're trying to say. You'd
> like to make the lab flexible maybe?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/avrgeek/avrgeek.github.io/pull/3/files#r37409123>.
>